docs/Logging/index.md

```diff
@@ -0,0 +1,77 @@
+# Logging
+
+The following table lists the possible log level values and it's corresponding function. Only information with a level equal to or higher than `CF_Log.Level` will be outputted to the `script.log` file.
```

Comment:
It would be nice if we could connect logging to a log sink which is either scripts.log or rpt, or a custom file, or whatever else someone might make, like an HTTP logger. 

We can do this as add-on later on, but maybe keep this in mind for now and see if it was possible to implement later without breaking existing usage.

Reply:
Create an issue for this

Reply:
#80